showcase/app/controllers/internationalization/translation.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Controller from '@ember/controller';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import { service } from '@ember/service';
+
+const SERVICE_TRANSLATIONS = {
+  title: {
+    key: 'showcase.pages.internationalization.translation.title',
+    default: 'Selected language (Fallback translation).',
+  },
+  body: {
+    key: 'showcase.pages.internationalization.translation.body',
+    default:
+      'The text is displayed in the selected language (Fallback translation).',
+  },
+  primaryAction: {
+    key: 'showcase.pages.internationalization.translation.actions.primary',
+    default: 'Primary action (Fallback translation)',
+  },
+  secondaryAction: {
+    key: 'showcase.pages.internationalization.translation.actions.secondary',
+    default: 'Secondary action (Fallback translation)',
+  },
+};
+
+export default class InternationalizationTranslationController extends Controller {
+  @service intl;
+  @service hdsIntl;
+
+  @tracked lang = null;
+
+  get serviceTranslations() {
```

Comment:
[suggestion] to make more explicit/evident what it is/what it refers to
```suggestion
  get hdsIntlTranslations() {
```